### DISCUSSION TITLE
Paseidon Template

### DISCUSSION THREAD

**Discussion by SmartSystemsTech** [primefaces/community#659] [PrimeFaces Templates]
Merhaba. Poseidon template yanlish yukleniyor.  template.xhtml resource goruyor(page source-dan gore biliyorum). Yardimci ola bilir misiniz?
![image](https://github.com/primefaces/community/assets/57983634/49ab70cb-70ce-41b7-9973-5122a5bd8a89)

**Comment by mertsincan** [comment]
Merhaba, console da herhangi bir hata alıyor musunuz? Son olarak hangi versiyonu kullanıyorsunuz?

**Comment by tandraschko** [comment]
i will close this for now
informations missing and please use english next time
